lib/Test2/Mock.pm

```diff
@@ -199,11 +199,12 @@ EOT
 package $class;
 #line $line "$file (Generated can)"
     sub {
+        use Scalar::Util 'reftype';
```

Comment:
This should go to the top of the file.

Reply:
I moved it to under the `package` line, assuming that's what you meant.